src/Document.js

```diff
@@ -187,6 +177,39 @@ export default class Document extends ParentNode {
         return null;
     }
 
+    get documentElement() {
+        return this._documentElement;
+    }
+
+    get head() {
+        let head = this._documentElement.getElementsByTagName('head').shift();
+        if (!head) {
```

Comment:
However, I do understand that, in practice, it could be annoying to manually add a body and a head